contracts/protocols/polygon/compound_iii/helpers.sol

```diff
@@ -0,0 +1,440 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+import "./interfaces.sol";
+import { DSMath } from "../../../utils/dsmath.sol";
+
+contract CompoundIIIHelpers is DSMath {
+    /**
+     *@dev Returns matic address
+     */
+    function getMaticAddr() internal pure returns (address) {
+        return 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    }
+
+    function getCometRewardsAddress() internal pure returns (address) {
+        return 0x45939657d1CA34A8FA39A924B71D28Fe8431e581;
+    }
+
+    function getConfiguratorAddress() internal pure returns (address) {
```

Comment:
I believe the configurator address is wrong here.
This is the implementation address.